contracts/src/utils/introspection/erc165.rs

```diff
@@ -16,11 +16,18 @@ use openzeppelin_stylus_proc::interface_id;
 #[interface_id]
 pub trait IErc165 {
     /// Returns true if this contract implements the interface defined by
-    /// `interface_id`. See the corresponding [ERC section]
-    /// to learn more about how these ids are created.
+    /// `interface_id`. See the corresponding [ERC section] to learn more about
+    /// how these ids are created.
```

Comment:
```suggestion
    /// `interface_id`. See the corresponding [ERC] to learn more about how
    /// these ids are created.
```
let's simplify this while we're at it